api-tester/src/main/java/com/revenuecat/apitester/java/EntitlementInfosAPI.java

```diff
@@ -2,14 +2,21 @@
 
 import com.revenuecat.purchases.EntitlementInfo;
 import com.revenuecat.purchases.EntitlementInfos;
+import com.revenuecat.purchases.VerificationResult;
 
 import java.util.Map;
 
-@SuppressWarnings({"unused"})
+@SuppressWarnings({"unused", "deprecation"})
 final class EntitlementInfosAPI {
     static void check(final EntitlementInfos infos) {
         final Map<String, EntitlementInfo> active = infos.getActive();
         final Map<String, EntitlementInfo> all = infos.getAll();
         final EntitlementInfo i = infos.get("");
+        final VerificationResult verification = infos.getVerification();
+    }
+
+    static void checkConstructor(Map<String, EntitlementInfo> all, VerificationResult verificationResult) {
```

Comment:
We should add these for all public models that have a public constructor, but we can do that in a different PR.